### DISCUSSION TITLE
how to preload a fbx file asset?

### DISCUSSION THREAD

**Discussion by tautologias345** [enable3d/enable3d#279] [Help / Questions]
I'm using as a Phaser 3 extension.

**Comment by yandeu** [comment]
Phaser's`preload()` is not async. Therefore, I would preload the fxb assets in another scene. Like a bootScene or preloadScene.

**Reply by tautologias345** [reply]
How can I do preload scene?

**Reply by yandeu** [reply]
See https://youtu.be/9ppKXYzDIuA

**Comment by tautologias345** [comment]
How can I preload fbx assets?

**Reply by yandeu** [reply]
See https://github.com/enable3d/enable3d-website/blob/67fed5ffddf98c068dc8c55b5b58aea624ad3c8b/src/examples/load-and-use-textures.html#L31
